.github/workflows/setup-cluster-resources/action.yaml

```diff
@@ -0,0 +1,34 @@
+name: Cluster Setup
```

Comment:
Is this file a duplicate? What is the difference here?

Reply:
Almost both files(.github/workflows/setup-cluster-resources/action.yaml and .github/workflows/integration-test-setup/action.yaml) are same the only diff is that `integration-test` file has `local disk` and in `setup-cluster-resources` file has `install deps`

Reply:
Please open an issue as a follow-up to see if we can remove this duplication further. Thanks

Reply:
#9131